action.js

```diff
@@ -144,25 +168,27 @@ async function addTaskToProject(client, taskId, projectId, sectionId) {
     if (!sectionId) {
         console.info('adding asana task to project', projectId);
         try {
-            return await client.tasks.addProjectForTask(taskId, {
-                project: projectId,
-                insert_after: null,
-            });
+            const body = {
+                data: {
+                    project: projectId,
+                },
+            };
+            const opts = {};
+            return await client.tasks.addProjectForTask(body, taskId, opts);
```

Comment:
```suggestion
            return await client.tasks.addProjectForTask(body, taskId);
```
No need for `opts` per [doc](https://developers.asana.com/reference/addprojectfortask). And same in the next changeset below.